### ISSUE TITLE
Example fails to build in us-gov-west-1

### ISSUE THREAD

**Issue by cyorobert** [stelligent/mu#348]
Hey, thanks for the hard work developing this project. I'm following along with https://ecsworkshop.com/platform/example/ and trying to get some infrastructure up and running on us-gov-west-1.

I've created the following configuration file:

mu.yml
```yml
---

environments:
  - name: acceptance
    provider: ecs
  - name: production
    provider: ecs
service:
    acceptance:
      disabled: true
    production:
      disabled: true
```

but receive the following error after running `mu env up acceptance && mu env up production`

```
ERROR    mu-{my-namespace}-iam-common:  CloudFormationRole (AWS::IAM::Role) CREATE_FAILED Partition "aws" is not valid for resource "arn:aws:iam::*:role/aws-service-role/ecs.amazonaws.com/AWSServiceRoleForECS*"
```

Is this an issue using Mu in govcloud, or an issue with my config?

**Comment by brentley** [comment]
looking at that error message, this makes me think that IAM behaves differently in govcloud than everywhere else:
`Partition "aws" is not valid for resource`

any chance you can swap to us-west-2 for educational purposes?

**Comment by cplee** [comment]
All the CloudFormation templates need to be updated to replace `arn:aws` with `arn:${AWS::Partition}`

**Comment by cplee** [comment]
fixed in #351
